README.md

```diff
@@ -53,6 +53,9 @@ OPTIONS:
     -t, --threshold <THRESHOLD>
             Sets the maximum amount of space to be used for Docker images (default: 10 GB)
 
+    -o, --older-than <OLDER THAN>
```

Comment:
How would you feel about calling this `-m, --min-age <DURATION>`?

Reply:
Good suggestion, the change has been made